main-serve-simple.py

```diff
@@ -17,34 +15,35 @@
 
 
 @ray.remote
-def log_frame(df: pd.DataFrame) -> DatasetProfile:
-    profile = DatasetProfile("")
-    profile.track_dataframe(df)
-    return profile
+def log_frame(df: pd.DataFrame) -> DatasetProfileView:
+    profile = DatasetProfile()
+    profile.track(df)
+    return profile.view()
 
 
 @serve.deployment()
 class Logger:
     def __init__(self) -> None:
-        # TODO how long will this local state stick around?
-        # TODO How many instances of Logger are actually "deployed"?
-        # TODO Can I pull state for all instances of Logger at some point to retrieve these profiles or do I need to coordinate that with some supervisor actor?
-        self.profile = DatasetProfile("")
+        # TODO having to make views like this is weird when the DatasetProfileView constructor exists
+        self.profile = DatasetProfile().view()
```

Comment:
@jamie256 Have an issue open for this https://github.com/whylabs/whylogs/issues/869

Reply:
You above code is the best, but I agree its awkward and a bit confusing, I would prefer we explicitly support the reduce seed case by providing a zero() static method? What you have works and is what the implementation of zero would do.